aspnet-core/xml/ns-Microsoft.AspNetCore.Mvc.Controllers.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Mvc.Controllers">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types to discover, initialize, and execute MVC controllers.</summary>
```

Comment:
```suggestion
    <summary>Provides types to discover, initialize, and run MVC controllers.</summary>
```
Per MS style guide:
run vs. execute
Avoid execute except to follow the UI. Use run instead. If the UI includes execute, the resulting action is still run.

Always use run in the context of operating systems, macros, and queries.